lua/mgn/stages/cl_overloading.lua

```diff
@@ -140,4 +145,4 @@ mgn.Stage.Overloading = {
 		mgn.CountdownMusic = nil
 		hudendtime = RealTime() + 2.1
 	end
-}
+}
```

Comment:
Please remove this change.

Reply:
i am bad with vscode and line endings idk how to fix